civihr.install

```diff
@@ -433,6 +438,7 @@ function _civihr_setup_users() {
   foreach ($users as $user) {
     _civihr_create_user($user['username'], $user['email'], $user['roles']);
   }
+
```

Comment:
Do we need this extra line here and the one before the `if (!variable_get('civihr_create_demo_users', false)) {` after the function definition?